tests/_fixtures/marathon-1-task/groups-sdk-services.json

```diff
@@ -1,56 +1,36 @@
 {
   "id": "/",
-  "dependencies": [
-
-  ],
+  "dependencies": [],
```

Comment:
the change in the file can be dropped to shrink down the size of the diff.

Reply:
those changes have been factored into separate commits, so i'd consider this well done.

also i'd vote for keeping it as it is, as else we'd waste the time of the next person touching the file, wouldn't we? :)

Reply:
I'll keep it, but extract the change into its own commit (like it should have been in the first place, my bad)